CMakeLists.txt

```diff
@@ -16,20 +17,59 @@ if (EXISTS "${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
     conan_basic_setup()
 endif()
 
-option(STD_C "Use the standard C library" ON)
+if(NOT DEFINED CPP_PLATFORM)
+    if(MSVC)
+        set(CPP_PLATFORM VisualCpp)
+    elseif(CMAKE_CXX_COMPILER_ID STREQUAL "IAR")
```

Comment:
I don't know if/how `if(IAR)` ever worked before. It has never been [provided by CMake](https://cmake.org/cmake/help/latest/manual/cmake-variables.7.html) as far as I can tell. CMake didn't learn how to [identify an IAR toolchain](https://cmake.org/cmake/help/v3.10/variable/CMAKE_LANG_COMPILER_ID.html?highlight=iar) until [version 3.10](https://cmake.org/cmake/help/latest/release/3.10.html).

Reply:
I found it. It was defined in the ewarm toolchain files.